website/content/commands/connect/envoy.mdx

```diff
@@ -227,6 +231,29 @@ This example assumes that the correct [environment variables](#api-options) are
 used to set the local agent connection information and ACL token, or that the
 agent is using all-default configuration.
 
+### Sidecar Proxy with ACLs Enabled
+
+In secure deployments, Consul's ACL system is enabled with a default `deny` policy.
+Nothing can access Consul API resources unless it presents a Consul ACL token with the
+necessary privileges. A sidecar proxy must have a Consul ACL token to act as itself and
+the service it fronts, and to access all that service's potential upstreams.
+The best way to generate such a token is to
+[use a service identity](/consul/commands/acl/token/create#create-a-token-for-a-service).
```

Comment:
```suggestion
To access Consul API resources, services and users must present a Consul ACL token with the
necessary privileges. A sidecar proxy configuration must also contain a Consul ACL token. This is so that the proxy and services that send traffic through the proxy can also access Consul API resources and access all of the services' potential upstreams.
We recommend creating a [service identity](/consul/commands/acl/token/create#create-a-token-for-a-service) when generating a token. Service identities enable you to quickly construct policies for services, rather than creating identical polices for each service.
```
Trying to be more specific than saying "nothing" can access resources. I also wan to be more concrete about what we mean in terms of a sidecar proxy "act[ing] as itself". Let me know if I am misinterpreting this information.

Reply:
I applied the spirit of this suggestion to some new wording (that I'll push to the repo in a few minutes) to preserve accuracy. Let me know if the new wording is clear and complies with our style.